### ISSUE TITLE
Model primary key is set to 0 after import with composite_primary_keys

### ISSUE THREAD

**Issue by guigs** [zdennis/activerecord-import#369]
````ruby
class Tag < ActiveRecord::Base
  self.primary_keys = :tag_id, :publisher_id
end

tags = [Tag.new(tag_id: 1, publisher_id: 1, tag: 'Mystery')]
Tag.import tags
tags.first

=> #<Tag:0x0055b6b1e5b010
 tag_id: 0,
 publisher_id: 1,
 tag: 'Mystery'>
````

Note that `tag_id` was set to `0`. In database the value was saved correctly.

I've investigated the cause, but don't know how to proper fix it.

The problem is that `ActiveRecord::Import::PostgreSQLAdapter#insert_many` is returning `ids` as `"(1,1)"`.

Then in `#set_ids_and_mark_clean` the line `model.id = id` will execute https://github.com/composite-primary-keys/composite_primary_keys/blob/master/lib/composite_primary_keys/base.rb#L106
that will call `CompositePrimaryKeys::CompositeKeys#parse` 
which expect id to be `"1,1"` (without parenthesis) https://github.com/composite-primary-keys/composite_primary_keys/blob/master/lib/composite_primary_keys/composite_arrays.rb#L30. So with `"(1,1)"` it will parse as `["(1", "1)"]`, which will be converted to `0` and `1`.

**Comment by jkowens** [comment]
Wow, good catch. I'm working on a fix (PR #371), but I'm wondering if there is a proper way to return composite ids on insert that ActiveRecord will understand.

I wrapped the RETURNING clause values with parentheses so that they would be returned as a group, otherwise we seem to only get the first key back?

https://github.com/zdennis/activerecord-import/blob/master/lib/activerecord-import/adapters/postgresql_adapter.rb#L48

So my solution as of right now is to just do `id.tr!('()', '')` before setting the id value.

**Comment by jkowens** [comment]
@guigs if you have a chance could you look at #371 and see if you can come up with any other options for fixing this issue? Thanks!

**Comment by jkowens** [comment]
@guigs the fix for this was just released in v0.17.1.
